exercises/concept/guessing-game/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+# Hints
+
+## General
+
+- [This page][pattern-matching] has a nice introduction to pattern matching in Haskell.
+
+## 1. Reply to a correct guess
+
+- You can use a constant pattern to match on a specific number.
+
+## 2. Reply to a close guess
+
+- You can use a constant pattern to match on a specific number.
+
+## 3. Reply to too low guesses
+
+- You can use a combination of a variable pattern and [guards][guards].
+
+## 4. Reply to too high guesses
+
+- You can use a combination of a variable pattern and [guards][guards].
+
+
+[pattern-matching]: http://www.learnyouahaskell.com/syntax-in-functions
```

Comment:
In accordance with https://github.com/exercism/haskell/pull/1104, we cannot use http://learnyouahaskell.com anymore. Please update this link to use https://learnyouahaskell.github.io instead.